### PR TITLE
Added Mediafire (Resolves #3022)

### LINKED ISSUE #3022: MediaFire | www.mediafire.com
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
     * Has a PRO subscription
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 

**What's the name of the service?** 
MediaFire: https://www.mediafire.com

> MediaFire is a file hosting, file synchronization, and cloud storage service based in Shenandoah, Texas, United States. Founded in June 2006 by Derek Labian and Tom Langridge, the company provides client software for Microsoft Windows, macOS, Linux, Android, iOS, BlackBerry 10, and web browsers. MediaFire has 43 million registered users and attracted 1.3 billion unique visitors to its domain in 2012.

**What should the Presence display?** 
It works similarly to the presence of MEGA, Google Drive among others for cloud storage and file sharing, then the presence can be similar to these.

**If possible, please provide a logo for the service (512x512 minimum)** 
image

### PR DESCRIPTION
**Pull request by DarkVillager**
Resolves #3022
mediafire6 2
mediafire 2 1
Mediafire 3 1
mediafire1 1
Mediafire4 1
Mediafire5 1

### CODE REVIEW

**Comment by DarkVillager** [comment]
(Resolves #3022)

File: websites/M/Mediafire/presence.ts [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    //     if (title.textContent.includes("Alle categorieën in")) {
+    //       title.textContent = title.textContent.replace("Alle categorieën in", "");
+    //     }
+    //     presenceData.details = "Viewing Category:";
+    //     presenceData.state = title.textContent;
+    //   } else if (r.test(page)) {
+    //     title = document.querySelector("#content > h1");
+    //     presenceData.details = "Viewing Category:";
+    //     presenceData.state = title.textContent;
+    //   }
```
Comment (ririxidev) [inline websites/M/Mediafire/presence.ts:129]: Delete commented statements

File: websites/M/Mediafire/presence.ts [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    } else if (page.includes("/press/inquiries")) {
+      title = document.querySelector("#content > h2");
+      presenceData.details = "Viewing:";
+      presenceData.state = title.textContent;
+    }
+    // if (page.includes("/software")) {
+    //   title = document.querySelector("#titlebar > div > h1")
+    //   presenceData.details = "Viewing:";
+    //   presenceData.state = title.textContent
+    // }
```
Comment (ririxidev) [inline websites/M/Mediafire/presence.ts:69]: Delete commented statements

**Review by ririxidev — CHANGES_REQUESTED** [review]
You should look at this:
~~ps. __Resolves__ thing must be in description of Pull request, not in comment nor title to get it working~~

File: websites/M/Mediafire/presence.ts (hunk 1 above)
Comment (DarkVillager) [reply websites/M/Mediafire/presence.ts:129]: Will do.

File: websites/M/Mediafire/presence.ts (hunk 2 above)
Comment (DarkVillager) [reply websites/M/Mediafire/presence.ts:69]: Will do.

**Comment by DarkVillager** [comment]
> You should look at this:
> 
> ~~ps. __Resolves__ thing must be in description of Pull request, not in comment nor title to get it working~~

Fixed

**Comment by ririxidev** [comment]
You already kinda "baked it". Because you added updates for another presence, but this pull request about Mediafire. So please, do seperate pull request for this update that you did. For now, I'm closing it till you will not fix it 😃

**Comment by DarkVillager** [comment]
> You already kinda "baked it". Because you added updates for another presence, but this pull request about Mediafire. So please, do seperate pull request for this update that you did. For now, I'm closing it till you will not fix it 😃

I hope it's fixed. Can you check for me? 👍

**Comment by ririxidev** [comment]
Still things from Markplaats

**Pull request closed without merging**